spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/config/EnableSpannerRepositories.java

```diff
@@ -97,7 +97,7 @@
 
 	/**
 	 * Configures the location of where to find the Spring Data named queries properties
-	 * file. Will default to {@code META-INFO/spanner-named-queries.properties}.
+	 * file. Will default to empty string.
```

Comment:
This does indeed default to `spanner-named-queries.properties`. I got it to work yesterday.
Probably safe to just revert the changes on this file, I fixed the `META-INFO` -> `META-INF` issue in another PR.